### PR TITLE
Native

### PR DESCRIPTION
**Pull request by cpcowart**
I tried to get this working but was having issues due to using dual setpoints. I implemented the range setpoints and moved all the property refresh i/o out of the selectors. I've been working with pykumo a little as well (I originally played with refactoring the blocking i/o in the module but decided that was an HA-specific requirement and therefore should probably be done in this project). I did start a small CLI demo (in pykumo) that I might work on a little more.

### CODE REVIEW

File: kumo/climate.py [hunk 1]
```diff
@@ -95,18 +160,43 @@ def temperature_unit(self):
...
+        self._hvac_mode = result
+
+    @property
+    def hvac_action(self):
+        """Return current hvac operation in action."""
+        return self._hvac_action
+
+    def _update_hvac_action(self):
+        """Refresh cached hvac action"""
+        standby = self._pykumo.get_standby()
```
Comment (dlarrick) [inline kumo/climate.py]: I'm going to need your mods to pykumo for this change

**Comment by dlarrick** [comment]
These changes look great, thanks! Just the one comment -- I'm going to need your get_standby change to pykumo

File: kumo/climate.py (hunk 1 above)
Comment (cpcowart) [reply kumo/climate.py]: Just sent a pull request with that method.

**Comment by dlarrick** [comment]
Thanks, I'm merging this. I will not get a chance to actually try it until tomorrow.

**Pull request merged**